routers/api/v1/api.go

```diff
@@ -1599,7 +1609,7 @@ func Routes() *web.Router {
 						Patch(reqToken(), reqRepoWriter(unit.TypeIssues, unit.TypePullRequests), bind(api.EditMilestoneOption{}), repo.EditMilestone).
 						Delete(reqToken(), reqRepoWriter(unit.TypeIssues, unit.TypePullRequests), repo.DeleteMilestone)
 				})
-			}, repoAssignment(), checkTokenPublicOnly())
+			}, repoAssignment())
```

Comment:
Why `checkTokenPublicOnly()` removed?

Reply:
it seems it was due to merge conflict, i have fixed it